src/test/scala/firrtlTests/features/LetterCaseTransformSpec.scala

```diff
@@ -125,8 +126,8 @@ class LetterCaseTransformSpec extends AnyFlatSpec with Matchers {
       { case ir.Circuit(_, _, "FOO") => true },
       { case ir.Module(_, "FOO",
                        Seq(ir.Port(_, "CLK", _, _), ir.Port(_, "RST_P", _, _), ir.Port(_, "ADDR", _, _)), _) => true },
-      /* Module "Bar" should be skipped via a ManipulateNamesBlocklistAnnotation */
-      { case ir.Module(_, "Bar", Seq(ir.Port(_, "OUT", _, _)), _) => true },
```

Comment:
Note: this test was actually broken before. This test is running `UpperCaseNames`, but it was consuming the block list annotation that was supposed to be sensitive to `LowerCaseNames`.